pinot-core/src/main/java/org/apache/pinot/core/transport/QueryRouter.java

```diff
@@ -118,6 +122,7 @@ public AsyncQueryResponse submitQuery(long requestId, String rawTableName,
       ServerRoutingInstance serverRoutingInstance = entry.getKey();
       ServerChannels serverChannels = serverRoutingInstance.isTlsEnabled() ? _serverChannelsTls : _serverChannels;
       try {
+        _serverRoutingStatsManager.recordStatsAfterQuerySubmission(requestId, serverRoutingInstance.getInstanceId());
```

Comment:
Apologies for asking again if you clarified this in the internal review. Shouldn't this be after line 126 ?

Reply:
Clarified with comments in code.